samples/logs.js

```diff
@@ -174,72 +177,72 @@ async function deleteLog(logName) {
 
 async function main() {
   require('yargs')
-    .demand(1)
-    .command(
-      'list',
-      'Lists log entries, optionally filtering, limiting, and sorting results.',
-      {
-        filter: {
-          alias: 'f',
-          type: 'string',
-          requiresArg: true,
-          description: 'Only log entries matching the filter are written.',
-        },
-        limit: {
-          alias: 'l',
-          type: 'number',
-          requiresArg: true,
-          description: 'Maximum number of results to return.',
-        },
-        sort: {
-          alias: 's',
-          type: 'string',
-          requiresArg: true,
-          description: 'Sort results.',
-        },
-      },
-      opts => {
-        listLogEntriesAdvanced(opts.filter, opts.limit, opts.sort);
-      }
-    )
-    .command('list-logs', 'Lists logs in your project.', {}, listLogs)
-    .command('list-simple <logName>', 'Lists log entries.', {}, opts =>
-      listLogEntries(opts.logName)
-    )
-    .command(
-      'write-simple <logName>',
-      'Writes a basic log entry to the specified log.',
-      {},
-      opts => {
-        writeLogEntry(opts.logName);
-      }
-    )
-    .command('delete <logName>', 'Deletes the specified Log.', {}, opts => {
-      deleteLog(opts.logName);
-    })
-    .example('node $0 list', 'List all log entries.')
-    .example(
-      'node $0 list -f "severity=ERROR" -s "timestamp" -l 2',
-      'List up to 2 error entries, sorted by timestamp ascending.'
-    )
-    .example(
-      'node $0 list -f \'logName="my-log"\' -l 2',
-      'List up to 2 log entries from the "my-log" log.'
-    )
-    .example(
-      'node $0 write my-log \'{"type":"gae_app","labels":{"module_id":"default"}}\' \'"Hello World!"\'',
-      'Write a string log entry.'
-    )
-    .example(
-      'node $0 write my-log \'{"type":"global"}\' \'{"message":"Hello World!"}\'',
-      'Write a JSON log entry.'
-    )
-    .example('node $0 delete my-log', 'Delete "my-log".')
-    .wrap(120)
-    .recommendCommands()
-    .epilogue('For more information, see https://cloud.google.com/logging/docs')
-    .help()
-    .strict().argv;
+      .demand(1)
```

Comment:
No need to do it right now, but we want to move away from `yargs` and towards having each snippet in its own file.    BigQuery has some great up to date examples:
https://github.com/googleapis/nodejs-bigquery/tree/master/samples

Reply:
Note for future